extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/i18n/hue_de.properties

```diff
@@ -0,0 +1,2 @@
+setup-flow.hue.error.1=Die Hue Bridge antwortet nicht. Bitte \u00FCberpr\u00FCfen Sie die angegebene IP-Adresse.
```

Comment:
I would prefer if we could use a more readable key like "bridge-unreachable" or "missing-ip-address-configuration"